frontend/src/components/visual-editor/form/inputs/message/QuickRepliesInput.tsx

```diff
@@ -24,10 +24,11 @@ type QuickRepliesInput = {
   minInput?: number;
 };
 
+
 const QuickRepliesInput: FC<QuickRepliesInput> = ({
   value,
   onChange,
-  minInput = 1,
+  minInput = 0,
```

Comment:
Changing default values directly can be risky if we're not sure how they're used across all cases. In this component, it's used in two scenarios: the "attachment" form needs `minInput = 0`, while the "quick replies" form needs `minInput = 1` . However, in this instance, the default value set here isn't necessary since `minInput` is always passed explicitly where the component is used. So adjusting it might not have any real impact.

Reply:
@medtaher123 thanks for the review, will fix it up